cristin-organization/src/test/java/no/unit/nva/cristin/organization/FetchCristinOrganizationHandlerTest.java

```diff
@@ -141,28 +140,6 @@ void shouldReturnBadRequestResponseOnMissingPathParam() throws IOException {
         assertThat(actualDetail, containsString(ERROR_MESSAGE_INVALID_PATH_PARAMETER_FOR_ID_FOUR_NUMBERS));
     }
 
-    @Test
-    void shouldReturnInternalServerErrorResponseOnUnexpectedException() throws IOException {
```

Comment:
this is generated by the framework. dont see any reason to explicit test it again